pkg/cmd/tests/options.go

```diff
@@ -35,6 +39,9 @@ var supportedBroadcastAddressTypes = []scyllav1.BroadcastAddressType{
 }
 
 type TestFrameworkOptions struct {
+	genericclioptions.ClientConfig
+	RunOptions
```

Comment:
Why is this reversing the existing inheritance?
https://github.com/scylladb/scylla-operator/pull/1824/files#diff-679d6ea01ce94872b44bde356e4eb82af41ca51988762be80625b957f8834a66L75

RunOptions are the top level entry point (named command) and contain an instance of TestFrameworkOptions (specific internal logic, not a named command).

Reply:
yep, should be other way around. fixed